scalableminds/webknossos
# Organization restricted datastore

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- set up local standalone datastore
- change db column `onlyAllowedOrganization` to random id => upload to that ds should fail
- change db column `onlyAllowedOrganization` to organization id => upload to that ds should work when logged in as user of that org

### Issues:
- fixes #4849 

------
